desktop/install/linux-install.md

```diff
@@ -268,6 +270,47 @@ Docker Desktop for Linux runs a Virtual Machine (VM) for the following reasons:
 
     As such, we have adjusted the default memory available to the VM in DD4L. You can tweak this setting to your specific needs by using the **Memory** slider within the **Settings** > **Resources** tab of Docker Desktop.
 
+## File sharing
+
+Docker Desktop for Linux uses [virtiofs](https://virtio-fs.gitlab.io/) as the
+default (and currently only) mechanism to enable file sharing between the host
+and Docker Desktop VM. In order not to require elevated privileges, without
+unnecessarily restricting operations on the shared files, Docker Desktop runs
+the file sharing service (`virtiofsd`) inside a user namespace (see
+`user_namespaces(7)`) with UID and GID mapping configured. As a result Docker
+Desktop relies on the host being configured to enable the current user to use
+subordinate ID delegation. For this to be true `/etc/subuid` (see `subuid(5)`)
+and `/etc/subgid` (see `subgid(5)`) have to be present. Docker Desktop only
+supports subordinate ID delegation configured via files. In case of both,
+`/etc/subuid` and `/etc/subgid`, Docker Desktop uses the first entry
+corresponding to the current user, to set up mappings for IDs above 0, and map
+the current users ID and GID to 0.
```

Comment:
```suggestion
supports subordinate ID delegation configured via files. Docker Desktop maps
the current user ID and GID to 0 in the containers. The mapping specified in
`/etc/subuid` and `/etc/subgid` are used for IDs above 0.
```

Reply:
You can have multiple entries per user, we'll only use the first one, that's important for the user to understand. I'll try to rephrase this.

Reply:
Ha ok, I didn't understand it this way.